### PR TITLE
example.md Finished

### PR DESCRIPTION
**Pull request by jdawgmad**
Finished example.md for the documentation

### CODE REVIEW

File: guide/Text/Embed/example.md [hunk 1]
```diff
@@ -1,3 +1,46 @@
...
+
+$title[Title;https://cdn.discordapp.com/icons/832255686783533066/f7131f694c6e1a2bd9c360d8b525d4e3.webp]
+$author[Custom Command;https://cdn.discordapp.com/icons/832255686783533066/f7131f694c6e1a2bd9c360d8b525d4e3.webp;https://cdn.discordapp.com/icons/832255686783533066/f7131f694c6e1a2bd9c360d8b525d4e3.webp]
+$image[https://cdn.discordapp.com/icons/832255686783533066/f7131f694c6e1a2bd9c360d8b525d4e3.webp]
+$description[Description]
+$thumbnail[https://cdn.discordapp.com/icons/832255686783533066/f7131f694c6e1a2bd9c360d8b525d4e3.webp]
+$color[#0c2238]
+$footer[Footer]
+
+## Embed Format
```
Comment (NervN) [inline guide/Text/Embed/example.md:15]: It's called Curl Format

File: guide/Text/Embed/example.md [hunk 2]
```diff
@@ -1,3 +1,46 @@
...
+## Function Format
+
+$title[Title;https://cdn.discordapp.com/icons/832255686783533066/f7131f694c6e1a2bd9c360d8b525d4e3.webp]
+$author[Custom Command;https://cdn.discordapp.com/icons/832255686783533066/f7131f694c6e1a2bd9c360d8b525d4e3.webp;https://cdn.discordapp.com/icons/832255686783533066/f7131f694c6e1a2bd9c360d8b525d4e3.webp]
+$image[https://cdn.discordapp.com/icons/832255686783533066/f7131f694c6e1a2bd9c360d8b525d4e3.webp]
+$description[Description]
+$thumbnail[https://cdn.discordapp.com/icons/832255686783533066/f7131f694c6e1a2bd9c360d8b525d4e3.webp]
+$color[#0c2238]
+$footer[Footer]
+
```
Comment (NervN) [inline guide/Text/Embed/example.md:14]: there's other missing ones, like $selectMenu, $button

File: guide/Text/Embed/example.md [hunk 3]
```diff
@@ -1,3 +1,46 @@
...
+{title:Title}
+{url:https#COLON#//cdn.discordapp.com/icons/832255686783533066/f7131f694c6e1a2bd9c360d8b525d4e3.webp}
+{footer:Footer}
+{description:Description}
+{color:#0c2238}
+{author:Custom Command:https#COLON#//cdn.discordapp.com/icons/832255686783533066/f7131f694c6e1a2bd9c360d8b525d4e3.webp}
+{thumbnail:https#COLON#//cdn.discordapp.com/icons/832255686783533066/f7131f694c6e1a2bd9c360d8b525d4e3.webp}
+{image:https#COLON#//cdn.discordapp.com/icons/832255686783533066/f7131f694c6e1a2bd9c360d8b525d4e3.webp}
+
```
Comment (NervN) [inline guide/Text/Embed/example.md:25]: This list is missing lot of things, check the curl reference page to populate it

File: guide/Text/Embed/example.md (hunk 2 above)
Comment (jdawgmad) [reply guide/Text/Embed/example.md:14]: hopefully got everything in next update

File: guide/Text/Embed/example.md (hunk 3 above)
Comment (jdawgmad) [reply guide/Text/Embed/example.md:25]: couldn't find curl reference page, so i hope theres enough

**Pull request merged**